packages/MarkdownEditor-Tests.package/MarkdownParagraphTest.class/instance/testBlankLineIsExtendableWithBlankLine.st

```diff
@@ -0,0 +1,6 @@
+tests - expandability
```

Comment:
In your tests you use the word `Extendable` instead of `Expandable`. I know, the meaning of the words are quite the same but I think, `tests - extendability` is the better name for the category.